src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+  const lines = sourceString.replace('\n', '').split(';');
+
+  for (let line of lines) {
+    line = line.trim();
+
+    if (!line.includes(':')) {
+      continue;
+    }
+
+    const [property, value] = line.split(':');
+
+    const cleanProperty = property.trim();
+    const cleanValue = value.replace(';', '').trim();
```

Comment:
The `replace(';', '')` is unnecessary here because the `split(';')` operation on line 10 already removes semicolons from the end of each line.

Reply:
Good point. Remove it